webui/src/Triggers/index.tsx

```diff
@@ -329,10 +329,10 @@ function TriggersTableRow({ controlId, item, editItem, moveTrigger, isSelected }
 			</td>
 			<td className="action-buttons">
 				<CButtonGroup>
-					<CButton color="white" onClick={doClone} title="Clone">
+					<CButton color="white" onClick={doClone} title="Clone" style={{ padding: 4 }}>
```

Comment:
I've pushed a change that should negate the need for these https://github.com/bitfocus/companion/commit/8fddb40ba120c1f228ec6fa58ca2e5f253c5f4e0

Reply:
Got it, I just reverted that.